### ISSUE TITLE
Crashed: com.apple.main-thread EXC_BREAKPOINT 0x0000000106a94984 

### ISSUE THREAD

**Issue by kinjalravel** [espresso3389/flutter_pdf_render#45]
Hi I'm experiencing the following crash only on ios. I'm unable to reproduce the issue but as far as I can tell the crash is not specific to one pdf

97% devices are iPhone and 90% has iOS 14  

```
Crashed: com.apple.main-thread
0  pdf_render                     0x106a94984 PdfPageTexture.updateTex(page:destX:destY:width:height:srcX:srcY:fullWidth:fullHeight:backgroundFill:) + 452 (SwiftPdfRenderPlugin.swift:452)
1  pdf_render                     0x106a97f10 specialized SwiftPdfRenderPlugin.updateTex(args:result:) + 341 (SwiftPdfRenderPlugin.swift:341)
2  pdf_render                     0x106a9931c specialized SwiftPdfRenderPlugin.handle(_:result:) + 4876 (<compiler-generated>:4876)
3  pdf_render                     0x106a90a78 @objc SwiftPdfRenderPlugin.handle(_:result:) + 76 (<compiler-generated>:76)
4  Flutter                        0x105e90e60 (Missing)
5  Flutter                        0x1059080d8 (Missing)
6  Flutter                        0x105c04f8c (Missing)
7  Flutter                        0x105ba5d94 (Missing)
8  Flutter                        0x105ba7bf4 (Missing)
9  CoreFoundation                 0x18c3a9fa0 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 32
10 CoreFoundation                 0x18c3a9ba0 __CFRunLoopDoTimer + 1064
11 CoreFoundation                 0x18c3a8ffc __CFRunLoopDoTimers + 328
12 CoreFoundation                 0x18c3a2ee4 __CFRunLoopRun + 1936
13 CoreFoundation                 0x18c3a221c CFRunLoopRunSpecific + 600
14 GraphicsServices               0x1a3f6c784 GSEventRunModal + 164
15 UIKitCore                      0x18ede2ee8 -[UIApplication _run] + 1072
16 UIKitCore                      0x18ede875c UIApplicationMain + 168
17 Educa Touch                    0x104169e1c main + 26 (main.m:26)
18 libdyld.dylib                  0x18c0626b0 start + 4

**Comment by espresso3389** [comment]
@kinjalravel Which version are you using? It seems very similar to #44 but it was already fixed on 1.0.11.

**Comment by kinjalravel** [comment]
I using ^1.0.9 , let me update it and see if it persist then i will reopen it with stack details
